server/routes/home.js

```diff
@@ -1,10 +1,24 @@
+const fetch = require('node-fetch')
+
 module.exports = {
   method: 'GET',
   path: '/',
   options: {
-    handler: (request, h) => {
+    handler: async (request, h) => {
```

Comment:
There are a couple of major issues with this handler.

1. `getData` should not directly manipulate the `data` variable. Altering variables from parent scopes makes the code more difficult to reason about and increases the likelihood of mistakes. A good alternative would be to return a promise which resolves to the fetched data.
2. Declaring `getData` inside the handler function is inefficient as the `getData` function will be recreated each time the handler is called. A reasonable alternative is to declare `getData` as a utility function in a separate file and import it at the top of this file. I'm not up to speed on Hapi yet but it appears that a Hapi-specific pattern might be to register a [server method](https://hapijs.com/api#-servermethods), perhaps via a [plugin](https://hapijs.com/api#plugins).

Reply:
Yes, it was a very quick hack using my rudimentary node skills. Please checkin a proper version when available. This was just so I could test out getting the containers talking to each other in Docker and K8.

Reply:
That's fine, I just wanted to get the issues documented before making changes. I'll make some changes.